$MirOS: ports/devel/sdl/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:45:28 tg Exp $
$OpenBSD: patch-configure_in,v 1.22 2003/12/07 19:45:43 espie Exp $
--- configure.in.orig	Sun Aug 28 06:30:56 2005
+++ configure.in	Sun Oct  2 11:15:20 2005
@@ -535,9 +535,6 @@ CheckNASM()
               win32)
                   NASMFLAGS="-f win32"
                   ;;
-              openbsd)
-                  NASMFLAGS="-f aoutb"
-                  ;;
               *)
                   NASMFLAGS="-f elf"
                   ;;
@@ -1082,9 +1079,6 @@ CheckOpenGL()
         AC_MSG_RESULT($video_opengl)
         if test x$video_opengl = xyes; then
             CFLAGS="$CFLAGS -DHAVE_OPENGL"
-            if test x$use_dlopen != xyes; then
-                AC_CHECK_LIB(dl, dlopen, SYSTEM_LIBS="$SYSTEM_LIBS -ldl")
-            fi
         fi
     fi
 }
@@ -1105,9 +1099,6 @@ CheckOpenGLQNX()
         if test x$video_opengl = xyes; then
             CFLAGS="$CFLAGS -DHAVE_OPENGL"
             SYSTEM_LIBS="$SYSTEM_LIBS -lGL"
-            if test x$use_dlopen != xyes; then
-                AC_CHECK_LIB(c, dlopen, SYSTEM_LIBS="$SYSTEM_LIBS", AC_CHECK_LIB(ltdl, dlopen, SYSTEM_LIBS="$SYSTEM_LIBS -lltdl"))
-            fi
         fi
     fi
 }
@@ -1251,7 +1242,7 @@ CheckPTHREAD()
             pthread_cflags="-I/usr/pkg/include -D_REENTRANT"
             pthread_lib="-L/usr/pkg/lib -lpthread -lsem"
             ;;
-        *-*-openbsd*)
+        *-*-openbsd* | *-*-ekkobsd* | *-*-mirbsd*)
             pthread_cflags="-D_REENTRANT"
             pthread_lib="-pthread"
             ;;
@@ -2096,7 +2087,7 @@ case "$target" in
         # NetBSD does not define "unix"
         CFLAGS="$CFLAGS -Dunix"
         ;;
-    *-*-openbsd*)
+    *-*-openbsd* | *-*-ekkobsd* | *-*-mirbsd*)
         ARCH=openbsd
         CheckDummyVideo
         CheckDiskAudio
@@ -2764,6 +2755,9 @@ if test "x$enable_rpath" = "xyes"; then
 else
   SDL_RLD_FLAGS=""
 fi
+if test $ARCH = openbsd -a x$enable_video_x11 = xyes; then
+  SDL_RLD_FLAGS="-L${X11BASE}/lib"
+fi
 
 case "$ARCH" in
   openbsd | netbsd | bsdi)
@@ -2797,7 +2791,7 @@ esac
 STATIC_SYSTEM_LIBS="$SYSTEM_LIBS"
 
 dnl Output the video drivers we use
-if test x$enable_video = xtrue; then
+if test x$enable_video = xyes; then
     if test "$VIDEO_SUBDIRS" = ""; then
         AC_MSG_ERROR(*** No video drivers are enabled!)
     fi
